.github/workflows/otf.yml

```diff
@@ -5,9 +5,9 @@
 name: OTF UATS
 
 on:
-  pull_request:
```

Comment:
keep it for pull request. You do have permissions to run the OTF tests, you just need to submit your pull request from this repository, not a fork

Reply:
fixed in 0f9e007583a54727976d6e7050bdeb6b2baa0f33